src/use-stepper.ts

```diff
@@ -204,6 +207,8 @@ function useStepper({
     const { onClick, ...otherButtonProps } = decrementProps;
     return {
       ...otherButtonProps,
+      'aria-hidden': true,
```

Comment:
I'm not sure `aria-hidden` is the right thing to do here. I don't see it mentioned in the spinbutton spec, but I would think it would need `aria-label` if not also `aria-controls`?